### PR TITLE
Drools 5456 - Drools Tools doesn't compile with JDK 11

### PR DESCRIPTION
**Pull request by gorky**
https://issues.redhat.com/browse/DROOLS-5456

<details>
<summary>
Updates to build under JDK 11.
</summary>
This PR makes the following updates to allow the Drools Projects to build under JDK 11:
* Updates version.org.eclipse.tycho to 2.3.0 to add support for JDK 11
* Updates the Maven Compile Goal to use the RELEASE nomenclature.
</details>

Jenkins run fdb

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -498,6 +498,17 @@
...
+    <!-- ************** -->
+    <!-- Build settings -->
+    <!-- ************** -->
+
+    <version.shade.plugin>3.2.4</version.shade.plugin>
+
+    <!-- maven-compiler-plugin -->
+    <maven.compiler.target>11</maven.compiler.target>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.release>11</maven.compiler.release>
```
Comment (mareknovotny) [inline pom.xml:511]: we need still to keep target for java 8, so this is not acceptable to be merged yet ;)

**Comment by mareknovotny** [comment]
ok to test

**Review by mareknovotny — CHANGES_REQUESTED** [review]
Hi @gorky, could you remove the properties for maven compiler plugin and we can merge this.

File: pom.xml (hunk 1 above)
Comment (gorky) [reply pom.xml]: Done.  Sorry about taking so long.

Comment (gorky) [reply pom.xml]: @mareknovotny Removed.

**Comment by mareknovotny** [comment]
thanks @gorky !

**Comment by mareknovotny** [comment]
jenkins do fdb

**Comment by mareknovotny** [comment]
closing as no activity here for an year

**Comment by gorky** [comment]
Sorry that I missed the original rebase request.  For latest activity, I was on the road, and then sick with Covid.  Updating now, will open a new PR.

**Pull request closed without merging**